pom.xml

```diff
@@ -93,12 +93,12 @@ THE SOFTWARE.
     <spotbugs.effort>Max</spotbugs.effort>
     <spotbugs.threshold>Medium</spotbugs.threshold>
 
-    <access-modifier.version>1.32</access-modifier.version>
-    <antlr.version>4.13.0</antlr.version>
-    <bridge-method-injector.version>1.27</bridge-method-injector.version>
+    <access-modifier.version>1.33</access-modifier.version>
+    <antlr.version>4.13.1</antlr.version>
+    <bridge-method-injector.version>1.29</bridge-method-injector.version>
```

Comment:
Why these seemingly unrelated changes?

Reply:
fixed with https://github.com/jenkinsci/jenkins/pull/8592/commits/89165aac9e56fef850375bedd674d59e658acdb2

Reply:
That had conflicts. I expect you improperly resolved them.

Reply:
These lines show as a non-conflicting change in the `git diff` during conflict resolution, perhaps you used that somehow?